src/main/java/org/quantumbadger/redreader/http/HTTPBackend.java

```diff
@@ -92,16 +92,10 @@ public static String encodeList(final List<PostField> fields) {
 	}
 
 	interface Listener {
+		void onError(@CacheRequest.RequestFailureType int failureType, Throwable exception, Integer httpStatus);
 
-		void onError(RequestFailureType failureType, Throwable exception, Integer httpStatus);
-
-		void onSuccess(
-				String mimetype,
-				Long bodyBytes,
-				InputStream body);
+		void onSuccess(String mimetype, Long bodyBytes, InputStream body);
```

Comment:
Yeah I'm really sorry about this, I've only started contributing on Github recently, and now know that auto-reformatting otherwise unchanged code isn't ideal, to say the least.